### PR TITLE
Add Prometheus metrics of Stackdriver sink performance

### PR DESCRIPTION
**Pull request by aleksandra-malinowska**
Metrics for monitoring Stackdriver sink performances: requests and timeseries sent count by result.

### CODE REVIEW

**Comment by aleksandra-malinowska** [comment]
@k8s-bot test this

**Comment by aleksandra-malinowska** [comment]
cc @crassirostris

File: metrics/sinks/stackdriver/stackdriver.go [hunk 1]
```diff
@@ -45,6 +48,8 @@ type metricMetadata struct {
 }
 
 var (
+	// metrics metadata
```
Comment (crassirostris) [inline metrics/sinks/stackdriver/stackdriver.go]: `Known metrics metadata` to be consistent with the capital letter + explain what metrics we're talking about

File: metrics/sinks/stackdriver/stackdriver.go [hunk 2]
```diff
@@ -110,6 +115,28 @@ var (
 		ValueType:  "INT64",
 		Name:       "container.googleapis.com/container/disk/bytes_total",
 	}
+
+	// prometheus metrics of sink performance
```
Comment (crassirostris) [inline metrics/sinks/stackdriver/stackdriver.go]: I would suggest rephrasing to `Sink metrics` or `Sink performance metrics`

File: metrics/sinks/stackdriver/stackdriver.go [hunk 3]
```diff
@@ -202,16 +229,30 @@ func CreateStackdriverSink(uri *url.URL) (core.DataSink, error) {
 		stackdriverClient: stackdriverClient,
 	}
 
+	// Register Prometheus metrics
```
Comment (crassirostris) [inline metrics/sinks/stackdriver/stackdriver.go]: `Register sink metrics` or `Register sink performance metrics` explain it better

**Review by crassirostris — CHANGES_REQUESTED** [review]
Do you have unit tests somewhere? If so, it would be great to test this

Code SGTM

File: metrics/sinks/stackdriver/stackdriver.go (hunk 1 above)
Comment (aleksandra-malinowska) [inline metrics/sinks/stackdriver/stackdriver.go]: Done

File: metrics/sinks/stackdriver/stackdriver.go (hunk 2 above)
Comment (aleksandra-malinowska) [inline metrics/sinks/stackdriver/stackdriver.go]: Done

File: metrics/sinks/stackdriver/stackdriver.go (hunk 3 above)
Comment (aleksandra-malinowska) [inline metrics/sinks/stackdriver/stackdriver.go]: Done

**Review by crassirostris — APPROVED** [review]

**Comment by piosz** [comment]
LGTM

**Pull request merged**